packages/core/src/components/cv-pagination/cv-pagination.vue

```diff
@@ -137,7 +137,7 @@ const newPagesArray = pageCount => {
   return Array.from({ length: pageCount }, (val, key) => key + 1);
 };
 
-const newFirstItem = (pageValue, pageSizeValue) => 1 + (pageValue - 1) * pageSizeValue;
+const newFirstItem = (pageValue, pageSizeValue) => Math.max(0, 1 + (pageValue - 1) * pageSizeValue);
```

Comment:
Not sure the Math.max(0,...) is needed in newFirstItem. Are you OK to remove it?

I realise it's been moved from numberOfItems() but pageValue and pageSizeValue should never be negative.

Reply:
I was ending up in a state where `pageValue = 0`, but I think that got fixed in another way. Will do.